common/types/mapper/thrift/shared.go

```diff
@@ -5063,6 +5063,7 @@ func ToStartWorkflowExecutionRequest(t *shared.StartWorkflowExecutionRequest) *t
 		Memo:                                ToMemo(t.Memo),
 		SearchAttributes:                    ToSearchAttributes(t.SearchAttributes),
 		Header:                              ToHeader(t.Header),
+		DelayStartSeconds:                   t.DelayStartSeconds,
```

Comment:
Please also update [proto IDLs](https://github.com/uber/cadence/tree/master/proto) and respective mappers as well. You may do it in a separate PR if you want.

Reply:
Sure, I can send another review for those changes